test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -1024,6 +1054,20 @@ class bcTestFixture {
     }
 };
 
+class bcInvalidTestFixture
```

Comment:
This class is almost identical to the one above, it looks like you could have a base class common for both to avoid code repetition. Just a suggestion.

Reply:
There is a plan for malicious block blockchain testsing, when the test tool will produce invlid block rlps and try to import. 
Thats for the future. for now just separate tests with invalid blocks that we have.